server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -102,14 +105,28 @@ func (w *Worker) work(ctx workflow.Context, revision string) error {
 	logger.Info(ctx, fmt.Sprintf("latest deployed revision %s", deployedRevision))
 
 	// TODO: fill in the rest
-
+	// Spin up a child workflow to handle Terraform operations
+	childWorkflowOptions := workflow.ChildWorkflowOptions{
+		TaskQueue: terraform.TaskQueue,
+		//TODO: match workflow id to format used by deploy workflow
+		WorkflowID: fmt.Sprintf("%s-%s", w.Repo.GetFullName(), w.Root.Name),
+	}
+	ctx = workflow.WithChildOptions(ctx, childWorkflowOptions)
+	terraformWorkflowRequest := terraform.Request{
+		Repo: w.Repo,
+		Root: w.Root,
+	}
+	err = workflow.ExecuteChildWorkflow(ctx, w.TerraformWorkflow, terraformWorkflowRequest).Get(ctx, nil)
```

Comment:
this is a cool idea, i wonder if this will actually work cuz im not sure how the whole reflection thing even works with this.